src/components/Articles.jsx

```diff
@@ -0,0 +1,37 @@
+import { useState, useEffect } from "react";
+import { fetchArticles } from "../utils/api";
+
+export default function Articles() {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    fetchArticles()
+      .then((response) => {
+        console.log(response, "<><>");
+        console.log(response.data, "<< article data in .then");
+        setArticles(response.data.articles);
+      })
+      .catch((err) => {
+        console.log("inside .catch");
+        console.log(err, "<< err");
+      });
+  }, []);
+
+  return (
+    <section>
+      <ul key="articles">
```

Comment:
Giving a key to the list item tag should be enough, but great that you thought about it and did it for the <li> tag.